apps/web/core/components/pages/projects/page-component.tsx

```
@@ -144,31 +164,26 @@
-						archivedAt: el.archivedAt,
-						isArchived: el.isArchived,
-						startDate: el.startDate,
-						endDate: el.endDate,
-						members: el.members,
-						managers: el.members,
-						teams: el.teams
-					}));
+		if (queries && Object.keys(queries).length > 0) {
+			console.log('first', queries);
+			getOrganizationProjects({ queries }).then((data) => {
+				if (data && data.items && data.items.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)